config/environments/production.rb

```diff
@@ -30,7 +30,7 @@
   # config.assets.css_compressor = :sass
 
   # Do not fallback to assets pipeline if a precompiled asset is missed.
-  config.assets.compile = false
+  config.assets.compile = true
```

Comment:
We're already precompiling assets as part of the production rollout, though. This is basically never what's wanted for production

Reply:
True, I was confused about what this actually did, I didn't realize at first that this compiles them on the fly if they don't exist.